js/components/App.js

```diff
@@ -24,6 +38,8 @@ export default Relay.createContainer(App, {
             node {
               id,
               name,
+              enabled,
+              ${ToggleWidgetEnableMutation.getFragment('widget')},
```

Comment:
Does Relay 0.2.1 (which I just got noticed was bumped in this repo in #27 ) allow some sort of shorthand version of the fragment here – or is that only for route queries?